integrationtest/src/main/scala/io/gearpump/integrationtest/minicluster/RestClient.scala

```
@@ -45,6 +48,28 @@ class RestClient(host: String, port: Int) {
     callFromRoot("version")
   }
 
+  def listMasters(): List[(String, Int)] = try {
+    queryMaster().cluster
+  }
+
+  def queryMaster(): MasterSummary = try {
+    val resp = callApi("master")
+    read[MasterData](resp).masterDescription
+  } catch {
+    case ex: Throwable => null
```

No need to wrap with `try`. Let it fail, if the result is not available